docs/src/getting_started/using_cmaize_as_build_system/cmake_like_project.rst

```diff
@@ -0,0 +1,174 @@
+*******************************************
+Example Project using CMake Style Structure
+*******************************************
+
+This chapter provides an overview of a project structured in a style that
+is common to CMake projects that uses CMaize as its build system.
+
+Project Overview
+================
+
+``MyCMakeProject`` is a simple C++ project that consists of the following
+components:
+
+* The ``TravelCostUtils`` library that contains some simple arithmetic
+  calculations for use in calculating travel costs.
+
+* The ``TravelCostCalc`` executable that calculates travel cost for a set of
+  predefined values using functions in ``TravelCostUtils``.
+
+* The ``Test`` directory contains some tests written using the Catch2 library
+  that test that ``TravelCostUtils`` works properly.
+
+Here is the file structure of the project:
+
+::
+
+    MyCMakeProject
+    ├── CMakeLists.txt
+    ├── cmake
+    │   └── get_cpp.cmake
+    ├── TravelCostUtils
+    |   ├── TravelCostUtils.cpp
+    |   └── TravelCostUtils.hpp
+    ├── TravelCostCalc
+    |   └── TravelCostCalc.cpp
+    └── Test
+        └── test.cpp
+
+This structure is commonly found in CMake projects. A subdirectory is created
+for each executable and library. A subdirectory is also created the project's
+tests.
+
+Creating the CMakeLists.txt File
+================================
+
+Here we will cover the contents of the ``CMakeLists.txt`` file at the root of
+the project.
+
+The Basics
+^^^^^^^^^^
+
+First we must start with the basics. We need to declare the minimum required
+version of CMake, the name and version of the project, and the C++ standard
+of the project.
+
+.. code-block:: cmake
+
+    # Declare minimum version required
+    cmake_minimum_required(VERSION 3.13.4)
+
+    # Declare project name and version
+    project(MyCMakeProject VERSION 1.0)
```

Comment:
You should also specify `LANGUAGES CXX` in `project` if your project is pure C++ (*i.e.* no C). The default is C and C++, which means you need a C compiler even though your project doesn't use it. Given that people will likely just copy your code we should strive to follow best practices.

Reply:
I added this to both example projects.